modules/tensor_mechanics/include/auxkernels/RadialDisplacementSphereAux.h

```diff
@@ -0,0 +1,46 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef RADIALDISPLACEMENTSPHEREAUX_H
+#define RADIALDISPLACEMENTSPHEREAUX_H
+
+#include "AuxKernel.h"
+
+class RadialDisplacementSphereAux;
+
+template<>
+InputParameters validParams<RadialDisplacementSphereAux>();
+
+class RadialDisplacementSphereAux : public AuxKernel
+{
+public:
+
+  /**
+   * Calculates the radial displacement for spherical geometries.
+   * Works for 3D, 2D axisymmetric, and 1D geometries
+   */
```

Comment:
This comment needs to be right before the class, not the constructor. (same with the other class)